com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ModelUtil.java

```diff
@@ -834,7 +859,20 @@ public static void main(String[] args) throws IOException {
             long timeStart = System.currentTimeMillis();
 
             InputStream is = new FileInputStream(file);
-            scene = loadScene(is, file.getPath(), new ModelImporter.Options());
+            byte[] bytes = IOUtils.toByteArray(is);
+
+            ModelImporter.FileDataResolver dataResolver = new ModelImporter.FileDataResolver();
+            scene = loadScene(bytes, file.getPath(), new ModelImporter.Options(), dataResolver);
+
+            // **********************************
+            for (ModelImporter.Buffer buffer : scene.buffers) {
+                if (buffer.buffer == null)
+                {
+                    System.out.printf("Unresolved buffer: %s\n");
+                }
+            }
```

Comment:
Our debug functionality code path (when invoking the ModelUtil as the main class via the test_model_util.sh)

Reply:
Maybe it should be a code comment?